### PR TITLE
Uploaded scroll & map, needs refreshing

### PR DESCRIPTION
**Pull request by devonpmack**

### CODE REVIEW

File: screens/GuitarScreen.js [hunk 1]
```diff
@@ -13,18 +21,35 @@ const styles = StyleSheet.create({
...
+  const scale = (num, in_min, in_max, out_min, out_max) => {
+    return ((num - in_min) * (out_max - out_min)) / (in_max - in_min) + out_min;
+  };
+
+  let index = 4;
+
+  const handleScroll = event => {
+    index = Math.floor(
+      scale(event.nativeEvent.contentOffset.y, 0, 2290, 0, song[0].line.length)
+    );
```
Comment (devonpmack) [inline screens/GuitarScreen.js]: Index needs to be a state variable and it should work.

`[index, setIndex] = useState(0)`

and then in this handler:
`setIndex(Math.floor(...))`

Comment (vinhvn) [reply screens/GuitarScreen.js]: Got it! thanks 👍

**Review by vincendiary — APPROVED** [review]
Looks good, thanks Dev :)

**Review by vincendiary — COMMENTED** [review]
Implemented useState, readjusted the scaling factor as well

**Pull request merged**